custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -126,7 +126,8 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
         {id: 7, name: 'ICDS-CAS Monthly Register'},
         {id: 8, name: 'AWW Performance Report'},
         {id: 9, name: 'LS Performance Report'},
-        {id: 10, name: 'Take Home Ration (THR)'}
+        {id: 10, name: 'Take Home Ration (THR)'},
+        {id: 11, name: 'Dashboard usage'},
```

Comment:
Hey, This change has to be under feature flag. This looks this is open now.

Reply:
added and verified